api/routes/refresh.js

```diff
@@ -20,6 +20,9 @@ module.exports = (app) => {
     if (session.roleArn.includes("aws-us-gov")) {
       Aws.config.update({region: 'us-gov-west-1'});
     }
+    else {
+      Aws.config.update({region: 'us-east-1'});
+    }
```

Comment:
```suggestion
		const region = session.roleArn.includes("aws-us-gov") ? 'us-gov-west-1' : 'us-east-1';
		Aws.config.update({region: region});
```

Now the `region` const is available to pass to `credentials.save` below without checking the `session.roleArn` again.